magento/values.yaml

```diff
@@ -0,0 +1,64 @@
+## Bitnami Magento image version
+## ref: https://hub.docker.com/r/bitnami/magento/tags/
+##
+imageTag: 2.0.5-r1
```

Comment:
This tag does not seem to exist in https://hub.docker.com/r/bitnami/magento/tags/